locales/zh.po

```diff
@@ -9569,6 +9569,10 @@ msgstr "使用安全连接(SSL)?"
 msgid "Additional JDBC connection string options"
 msgstr "额外的JDBC连接字符串选项"
 
+#: src/metabase/driver/common.clj target/classes/metabase/driver/common.clj
+msgid "Snowflake hostname (in case of privatelink or reverse proxy)"
+msgstr "snowflake主机名（如果是PrivateLink或反向代理"
```

Comment:
Please don't add translations in PRs like this. These files are generated automatically from [POEditor](https://poeditor.com/join/project/ynjQmwSsGh). Go add the translations there if you want after this PR is merged.